02-starting-with-data.Rmd

```diff
@@ -66,7 +66,7 @@ surveys <- read.csv("data/portal_data_joined.csv")
 
 This statement doesn't produce any output because, as you might recall,
 assignments don't display anything. If we want to check that our data has been
-loaded, we can print the variable's value: `surveys`.
+loaded, we can see the contents of the data frame by typing its name: `surveys`.
```

Comment:
Has the term "data frame" been used at this point in the lesson? I think not, right? If not, maybe we shouldn't use it here yet.

Reply:
Yes, the section starts with a sentence containing "...we will use read.csv() to load into memory the content of the CSV file as an object of class data.frame". 
An alternative could be to add "object", as in "we can see the contents of the data frame object by typing its name: `surveys`."